app/models/access_token.rb

```diff
@@ -5,6 +5,10 @@ class AccessToken < ApplicationRecord
 
   serialize :scopes, Array
 
+  audited only: %i[owner_id scopes name permission created_at updated_at]
+
+  delegate :provider_id_for_audits, to: :owner
```

Comment:
`owner` is a polymorphic association of `AccessToken`, which now seems to only have `User` as type in the db of Saas. Just in case, unless or until we are 100% positive that no other type of entity can own an access token, maybe better start with:

```suggestion
  def provider_id_for_audits
    owner.try(:provider_id_for_audits)
  end
```

Reply:
Look at this (the `class_name`):
https://github.com/3scale/porta/blob/e6be7e4c686590fc7f537f723ad6567f8b709298/app/models/access_token.rb#L2

And this:
![image](https://user-images.githubusercontent.com/11318903/73067747-c42e8680-3ea9-11ea-822f-43fe9ce74daa.png)

Reply:
I don't know why it's polymorphic then 🤷‍♂️

Reply:
Me neither :joy: It is just overcomplicating :thinking: Maybe in the past, it needed to be polymorphic for real but not anymore :woman_shrugging: